libs/blocks/modal/modal.js

```diff
@@ -152,9 +152,11 @@ export async function getModal(details, custom) {
   const focusVisible = { focusVisible: true };
   const focusablesOnLoad = [...dialog.querySelectorAll(FOCUSABLES)];
   const titleOnLoad = dialog.querySelector('h1, h2, h3, h4, h5');
+  const isVideoFragment = dialog.querySelector('.fragment:first-child')?.dataset?.path?.includes('modals/videos');
```

Comment:
Should a modal know about all the content it can have, e.g. a video fragment?

Reply:
There are two scenarios for the focusing functionality inside a dialog.
1)dialog modal is aware of all the contents inside it and here we focus on the first focusable element. eg:geo routing pop up.
2)dialog is unaware of its contents during initialization, any dialog that contains an iframe inside it. eg:commerce modal, video modal.
The code would be much simpler if all dialog were of the first type, since we would be focusing on the first focusable element but there's a need for us to handle this specific case because the fragment url is being considered as a focusable element since it is an anchor tag in milo.This anchor is converted into an iframe with a video tag and leaving nothing for us to focus on.